architecture-decision-records/adr-018-Knative_based_Kyma_eventing.md

```diff
@@ -0,0 +1,45 @@
+# ADR 018: Knative-based Kyma Eventing 
+
+Created on 2018-08-19 by Ahmed Abdalla (@Abd4llA).
+
+## Context
+
+Based on the decision of building Kyma on top of Knative and along the effort to reuse as much of knative functionality as possible, the eventing area shines as a good area for alignment. That is why a decision was made to have Kyma leverage as much of knative eventing as possible.This is a an architecture decision on how to achieve that target.
```

Comment:
1. `Based on the decision of building Kyma on top of Knative, and` ...
2. Capitalize Knative and Knative Eventing
3. as possible.This <- space needed here